src/components/02_atoms/MultipleFields/MultipleFields.js

```diff
@@ -171,13 +171,15 @@ class MultipleFields extends Component {
       state: { newItemAdded },
       props: { label, value: values, component, onChange },
     } = this;
-    const usedValues = values.length ? values : [this.createEmptyItem()];
+    const usedValues = (values && values.length && values) || [
```

Comment:
`values && values.length && values` needed to be added in order to return `values` instead of `values.length` 

`values && values.length` returns `values.length`

Reply:
why would you need both values && values twice

Reply:
oh I see what you did, mh, at least I was able to understand 1. on the first read

Reply:
first `values` to validate if the object is null or not, 
then `values.length` to check if the array is empty or not
lastly 'values' is to return the value as `values` instead of the `values.length`

if it would have been `values && values.length`, it would return the value as `values.length` instead of the actual object which is `values`

Reply:
Well, I totally understand how it works, no worries. The problem is someone new to the project trying to understand the code. It feels like 1. which is a bit more explicit would be the nicer alternative overall, but that's maybe just me.

Reply:
This is a clever solution. Nice job @vishalchandna1. However. I think that the first solution is probably a bit more explicit and easier to understand. Perhaps we could add just a bit of documentation about what is happening, and then sticking with the current solution seems good.

Reply:
Good idea!

Reply:
Got it, I will add it right away.